File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/Effluente.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class Effluente : BaseCodeDescr
    {
        public UnitaDiMisura udm { get; set; }
        public TipoEffluente tipoEffluente { get; set; }
        public decimal carico { get; set; }
        public decimal N { get; set; }

        public Effluente(int codice) : base(codice, "")
        {

        }


        public Effluente() : base() { }

    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/Effluente.ts
import { BaseCodeDescr } from "../baseClass/baseCodeDescr";
import { TipoEffluente } from "./TipoEffluente";
import { UnitaDiMisura } from "./UnitaDiMisura";

export class Effluente extends BaseCodeDescr {

    udm: UnitaDiMisura;
    tipoEffluente: TipoEffluente;
    carico: number;
    N: number;

    constructor(codice: number) {
        super(codice);
    }

}
